index.php

```diff
@@ -44,7 +44,11 @@
         $data['content']['bookingChannel'] = $booking->channel_id;
         $data['content']['bookingId'] = $booking->booking_id;
 
-    } elseif (isset($_POST['postRequesBooking'])) {
+    } elseif (isset($_POST['postDeleteTemporalBooking'])) {
+        $booking = $reserSys->deleteBooking($_POST['postRequestedBooking'], $_POST['postChannelId']);
+        $data['content']['bookingError'] = true;
```

Comment:
Why is true?

Reply:
What are you doing with booking object? is there anything checked? I dont think so

Reply:
'bookingError' is not the best name, it should be something more around 'reservationDeleted'. It's a boolean value so that I can show the correct mustache template just by checking it.

I'm not doing anything with the booking object, i'll take it out of the method response, unless i implement the error handler